ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -379,6 +375,26 @@ export const WorkspaceEdit = fp.flow(
       };
     }
 
+    initializeCdrVersions(props: WorkspaceEditProps) {
+      const defaultValue = this.getCdrVersions(DEFAULT_ACCESS_TIER);
+      if (this.isMode(WorkspaceEditMode.Create)) {
```

Comment:
Is it better to combine this "if condition" with line 383 somehow. for example 

```
if(this.isMode(...) || !props.workspace) 
     return defaultValue;
return this.getCdrVersions(props.workspace.accessTierShortName)
```

Reply:
Agreed, this is clearer. Done 👍